game_eggs/steamcmd_servers/subsistence/egg-subsistence-wine.json

```diff
@@ -0,0 +1,50 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2021-12-20T02:19:34-05:00",
+    "name": "Subsistence Wine",
+    "author": "th3dilli@gmx.at",
+    "description": "a Subsistence Wine egg",
+    "features": null,
+    "images": [
+        "quay.io\/parkervcp\/pterodactyl-images:debian_wine-5"
```

Comment:
Please migrate to our new docker images https://github.com/parkervcp/yolks#wine

Reply:
i did try today but ran in to some issues

will take another look at it soon